bears/vcs/git/GitCommitBear.py

```diff
@@ -34,14 +34,22 @@ class GitCommitBear(GlobalBear):
             'issue': r'(?:\w+/\w+)?#(\S+)',
             'full issue': r'https?://gitlab\S+/issues/(\S+)',
         },
+        'bitbucket': {
+            'issue': r'#(\S+)',
+            'full issue': None,
+        },
     }
     SUPPORTED_HOST_KEYWORD_REGEX = {
         'github': (r'[Cc]lose[sd]?'
                    r'|[Rr]esolve[sd]?'
                    r'|[Ff]ix(?:e[sd])?'),
         'gitlab': (r'[Cc]los(?:e[sd]?|ing)'
                    r'|[Rr]esolv(?:e[sd]?|ing)'
-                   r'|[Ff]ix(?:e[sd]|ing)?')
+                   r'|[Ff]ix(?:e[sd]|ing)?'),
+        'bitbucket': (r'[Cc]los(?:e[sd]?|ing)'
+                      r'|[Rr]esolv(?:e[sd]?|ing)'
+                      r'|[Ff]ix(?:e[sd]|ing)?'
+                      r')(?:(?:[ \t]*(?:bug|issue|ticket)?)?'),
```

Comment:
The shortlog should say something like 'Support bitbucket type issue references' or something along those lines since you are specifically aiming to improve those aspects of the bear. The commit message could also be improved by removing the example.